torch/csrc/jit/script/schema_type_parser.cpp

```diff
@@ -1,11 +1,31 @@
 #include <torch/csrc/jit/script/schema_type_parser.h>
 #include <ATen/core/interned_strings.h>
-#include <torch/csrc/jit/alias_info.h>
-#include <torch/csrc/jit/ir.h>
+#include <ATen/core/alias_info.h>
+#include <ATen/core/jit_type.h>
 #include <torch/csrc/jit/script/lexer.h>
 #include <torch/csrc/jit/script/parse_string_literal.h>
 #include <string>
 
+using c10::Symbol;
+using c10::GeneratorType;
+using c10::IntType;
+using c10::DeviceObjType;
+using c10::NumberType;
+using c10::StringType;
+using c10::BoolType;
+using c10::NoneType;
+using c10::FloatType;
+using c10::OptionalType;
+using c10::TupleType;
+using c10::TensorType;
+using c10::DimensionedTensorType;
+using c10::CompleteTensorType;
+using c10::FutureType;
+using c10::DictType;
+using c10::ListType;
+using c10::VarType;
+using c10::AliasInfo;
```

Comment:
I feel like this is going to end up wasting people's time.  For people who haven't done it before, they are going to take 5 minutes to figure out why, despite being in the torch namespace, they need to import all the types here. Can we put these in one place?

Reply:
What do you propose? Putting `using` statements into header files is a strong code smell and needs a very good justification. We did it for including `c10` into `at`, but that was only done because we were forced to do it for bc reasons. I don't think we should also include `c10` into `torch`.

Reply:
Are you proposing that TorchScript's dictionary type should be c10::DictType and not torch::DictType? That is insanely confusing for contributors.

Reply:
So we already have auto-export of everything c10:: in at:: and caffe2::. Shall we also auto-export to torch::? (it aligns with the plan to have one namespace to rule them all)